clients/extension/scripts/popup.js

```diff
@@ -80,32 +82,36 @@ function find(query, context = document) {
 
 /**
  * @param {Error} err
- * @return {HTMLAnchorElement}
+ * @return {HTMLButtonElement}
  */
 function buildReportErrorLink(err) {
   const issueBody = `
 **Lighthouse Version**: ${getLighthouseVersion()}
 **Lighthouse Commit**: ${getLighthouseCommitHash()}
 **Chrome Version**: ${getChromeVersion()}
 **Initial URL**: ${siteURL}
-**Error Message**: ${err.message}
+**Error Message**: ${err.friendlyMessage || err.message}
 **Stack Trace**:
 \`\`\`
 ${err.stack}
 \`\`\`
     `;
 
-  const url = new URL('https://github.com/GoogleChrome/lighthouse/issues/new');
-
-  const errorTitle = err.message.substring(0, MAX_ISSUE_ERROR_LENGTH);
-  url.searchParams.append('title', `Extension Error: ${errorTitle}`);
-  url.searchParams.append('body', issueBody.trim());
-
-  const reportErrorEl = document.createElement('a');
+  const reportErrorEl = document.createElement('button');
   reportErrorEl.className = 'button button--report-error';
-  reportErrorEl.href = url.href;
-  reportErrorEl.textContent = 'Report Error';
-  reportErrorEl.target = '_blank';
+  reportErrorEl.textContent = 'Copy to clipboard';
```

Comment:
at the first glance, the current copy suggests it would only copy the provided error message to the clipboard. Not obvious there are more details if only you copy to the clipboard.

"Copy details to clipboard"?

Reply:
I like `Copy details to clipboard` (or something like it) as well (do we also want a copy to cliboard icon?)

Reply:
Fixed in https://github.com/GoogleChrome/lighthouse/pull/8294/commits/84bac693c0ce0d722bd3bf2fa95838afcab258b2